doc/user-manual/tools/command-line-options.rst

```diff
@@ -785,6 +785,15 @@ Pattern matching and equality
 
      Default, opposite of :option:`--flat-split`.
 
+.. option:: --polarity, --no-polarity
+
+     .. versionadded:: 2.6.5
```

Comment:
Minor nitpick: the next version will likely be 2.7 (but we can also update this later)